src/lib.rs

```diff
@@ -95,6 +95,8 @@
 //!
 //!
 #![allow(bare_trait_objects)]
+// Required for rustc 1.29
+#![recursion_limit = "128"]
```

Comment:
I don't know how serious is this issue. In the next commit, our current implementation of match_token requires this. It is possible to get around this, but in the interest of time I have decided to ignore this

Reply:
lol!! Looks good to me.

For future reference this is because of the massive `match_tokens!` invocation in cov.rs